portugues/revisa/tenta-nao-consegue/amigavel/amigavel.md

```diff
@@ -0,0 +1,2 @@
+O desenvolvedor te envia um e-mail dizendo que seria muito importante se você conseguisse reproduzir o bug novamente. E ele até
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.